app/frontend/translations.ts

```diff
@@ -3,168 +3,134 @@ import debugLog from './helpers/debugLog'
 import {ADALITE_CONFIG} from './config'
 import {LEDGER_VERSIONS, TREZOR_VERSIONS} from './wallet/constants'
 import {Lovelace, CryptoProviderFeature} from './types'
+import {ExternalErrorReason, InternalErrorReason} from './errors'
 
 const {ADALITE_MIN_DONATION_VALUE} = ADALITE_CONFIG
 
 const ledgerTroubleshootingSuggestion =
   'If you are using Ledger, please make sure Ledger Live app is closed and try connecting your device using the "Connect with WebUSB" functionality (the button underneath "Unlock with Ledger"). For more information please read the section concerning Ledger in our troubleshooting suggestions.'
 
-const translations = {
-  SendAddressInvalidAddress: () => 'Invalid address',
-  SendAmountIsNan: () => 'Invalid format: Amount has to be a number',
-  SendAmountIsNotPositive: () => 'Invalid format: Amount has to be a positive number',
-  SendAmountInsufficientFunds: ({balance}) =>
+const translations: {[key in InternalErrorReason]: (params?: any) => string} = {
```

Comment:
do we even have translations of something else than errors? If not, it shouldn't be called "translations"

Reply:
agree, something like `errorReasons` sounds more suitable

Reply:
`const errorMessages`?

Reply:
ah sorry, I see "ErrorReason" is already used, so errorMessages perhaps